docs/src/testing/test-attributes.md

```diff
@@ -0,0 +1,137 @@
+## Test Attributes
+
+`snforge` allows setting test attributes for test cases in order to modify their behavior.
+
+Currently, those attributes are supported:
+
+- `#[test]`
+- `#[ignore]`
+- `#[should_panic]`
+- `#[available_gas]`
+- `#[fork]`
+- `#[fuzzer]`
+
+### `#[test]`
+
+Marks the function as a test case, to be visible for test collector.
+Read more about test collection [here](./test-collection.md).
+
+### `#[ignore]`
+
+Marks the function as ignored, it will be skipped after collecting.
+Use this if you don't want the test to be run (the runner will display how many tests were ignored in the summary).
+
+Read more about the behavior and how to override
+this [here](./testing.md#ignoring-some-tests-unless-specifically-requested).
+
+### `#[should_panic]`
+
+A test function can be marked with this attribute, in order to assert that the test function itself will panic.
+If the test panics when marked with this attribute, it's considered as "passed".
+
+Moreover, it can be used with either a tuple of shortstrings or a string for assessment of the exit panic data
+(depending on what your contract throws).
+
+#### Usage
+
+Asserting the panic data can be done with multiple types of inputs:
+
+ByteArray:
+
+```rust
+#[should_panic(expected: "No such file or directory (os error 2)")]
+```
+
+Shortstring:
+
+```rust
+#[should_panic(expected: 'panic message')]
+```
+
+Array of shortstrings:
```

Comment:
To be more precise - shouldn't it be `Tuple of shortstrings`?

Reply:
Yes, it should